packages/create-svelte/templates/default/package.json

```
@@ -13,7 +13,7 @@
 		"@sveltejs/kit": "workspace:*",
 		"svelte": "^3.44.0",
 		"svelte-preprocess": "^4.9.8",
-		"typescript": "^4.4.3"
+		"typescript": "^4.5.5"
```

```suggestion
		"typescript": "~4.5.5"
```

Might be good to recommend it here too.

---

done